web/src/features/fireWeather/slices/parseModelValuesHelper.ts

```diff
@@ -8,10 +8,15 @@ export const parseModelValuesHelper = (
   const pastModelValues: ModelValue[] = []
   const modelValues: ModelValue[] = []
   const noonModelValues: ModelValue[] = []
-  const allModelValues = model_runs.reduce(
-    (values: ModelValue[], modelRun) => values.concat(modelRun.values),
-    []
-  )
+  const reducer = (values: ModelValue[], modelRun: ModelRun) => {
```

Comment:
Quick question about the response structure, why do we have them this way?

Reply:
Oh is it because the model timestamp can vary for individual model values?

Reply:
The structure from the API is
-model run
--predictions

It didn't make sense to repeat the model run information on every single prediction.